docs/readthedocs/source/doc/Nano/QuickStart/pytorch_onnxruntime.md

```diff
@@ -0,0 +1,77 @@
+# BigDL-Nano PyTorch ONNXRuntime Acceleration Quickstart
+
+**In this guide we will describe how to apply ONNXRuntime Acceleration on inference pipeline with the APIs delivered by BigDL-Nano in 4 simple steps**
+
+### **Step 0: Prepare Environment**
+We recommend using [conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/) to prepare the environment. Please refer to the [install guide](../../UserGuide/python.md) for more details.
+
+```bash
+conda create py37 python==3.7.10 setuptools==58.0.4
+conda activate py37
+# nightly bulit version
+pip install --pre --upgrade bigdl-nano[pytorch]
+# set env variables for your conda environment
+source bigdl-nano-init
+```
+
+Before you start with onnxruntime accelerator, you are required to install some onnx packages as follows to set up your environment with ONNXRuntime acceleration.
+```bash
+pip install onnx onnxruntime
+```
+### **Step 1: Load the data**
+```python
+from torchvision.io import read_image
+from bigdl.nano.pytorch.vision import transforms
+
+paths = ["../Image/cat.jpg", "../Image/dog.jpg"]
+data_transform =  transforms.Compose([transforms.RandomCrop(224),
```

Comment:
You should not use data augmentation in the inference pipeline.
Maybe you should use the "weights.transforms()" here (https://pytorch.org/vision/stable/models.html#using-the-pre-trained-models)